<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>792</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>792</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>download.ico</normaloff>download.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>다운로드 위치</string>
    </property>
    <widget class="QLineEdit" name="locaton_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>701</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="location_button">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>검색어 입력</string>
    </property>
    <widget class="QLineEdit" name="swordEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>검색 리스트</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>751</width>
       <height>361</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>550</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>선택항목 다운로드</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>도움말</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homepage_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Radibrary 홈페이지 이동</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
